README.md

```diff
@@ -26,8 +26,8 @@ Click on an SVG to preview it:
 1. Drop the SVGs into one of the vector folders.
     - Icons specific to Red Hat concepts should have a `rh-` prefix and belong in the `src/iconfont/vectors/rh_icon` folder
     - Universal web interface icons need a `web-` prefix and belong in the `src/iconfont/vectors/web_icon` folder.
-2. Run `grunt` to update the font, or `grunt watcher` to update and preview.
-    - Note: the SVG files are parsed alphabetically, so if you add new icons somewhere besides the bottom of the list, they will cause unicode values to change on existing icons that are alphabetically lower in the list.  This is why we should *never* be calling an icon by it's unicode value via `content`.
+2. Delete the two codepoints map files: `codepoints.rh_web_icon.map` and `codepoints.rh_icon.map`. These will be recreated when you run `grunt` or `grunt watcher`
```

Comment:
Should this not just be added to the clean task?

Reply:
I added that here: https://github.com/RedHatOfficial/rh-iconfont/pull/11

Reply:
Cool cool. I'll close this then 👍